src/App.js

```diff
@@ -1,11 +1,24 @@
+/* eslint max-len: ["error", { "code": 1000 }] */
 import './App.css';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './pages/Home';
+// import Calc from './pages/Calc';
+import Quote from './pages/Quote';
+import Nav from './components/Navbar/Navbar';
 
 function App() {
   return (
-    <div className="App">
-      <Calculator />
-    </div>
+    <BrowserRouter>
+      <div className="App">
+        <Nav />
+        <Routes>
+          <Route path="/home" exact element={<Home />} />
+          <Route path="/quote" element={<Quote />} />
+          <Route path="/calculator" element={<Calculator />} />
+        </Routes>
```

Comment:
- [ ] The Home component should be rendered on the root route. Right now if users will open your website they will see a blank page they need to click on the Home button to go to the homepage. Please fix this issue and make sure you call your Home component on the root route.
  - To fix this you can consider using this code `<Route path="/" exact element={<Home />} />`